modules/core/src/helpers/__mocks__/deployment.config.json

```diff
@@ -50,6 +50,7 @@
         "appName": "Console",
         "appTitle": "WSO2 Identity Server Console",
         "appWhiteLogoPath": "/assets/images/branding/logo-inverted.svg",
+        "applicationSharingStatusPollingInterval": 5000,
```

Comment:
Since we poll the same endpoint in all async operations, can't we generalize this config to something like asyncOperationStatusPollingInterval?